p2p/src/authenticated/actors/dialer.rs

```diff
@@ -24,7 +27,7 @@ use tracing::debug;
 
 pub struct Config<C: Scheme> {
     pub registry: Arc<Mutex<Registry>>,
-    pub connection: connection::Config<C>,
+    pub connection: ConnectionConfig<C>,
```

Comment:
Agree that this is probably a good switch (from using `connection::Config` and importing `connection::self`)